src/providers/google/shared.ts

```diff
@@ -62,3 +64,12 @@ export const CHAT_MODELS = [
   'medlm-medium',
   'medlm-large',
 ];
+
+export const CHAT_MODEL_ALIASES: Record<string, string> = {
+  'gemini-flash-latest': 'gemini-2.5-flash-preview-09-2025',
+  'gemini-flash-lite-latest': 'gemini-2.5-flash-lite-preview-09-2025',
+};
+
+export function resolveChatModelName(modelName: string): string {
+  return CHAT_MODEL_ALIASES[modelName] ?? modelName;
+}
```

Comment:
Google should do this by default - I don't want to have to maintain this. Can you please verify if this is necessary? If they don't do it for whatever the reason we should still remove it.